config.json.sample

```diff
@@ -0,0 +1,20 @@
+{
+    "gutenberg_illustrated_binary_path" : "",
+    "data_directory" : "",
+    "policies" : {
+        
+    },
+    "integrations" : {
+    	"Postgres" : {
+    	    "production_url" : "postgres://simplified:61%mJO@localhost:5432/simplified_content_dev",
```

Comment:
The production URL isn't necessary to run the unit tests, so you should be able to remove this bit.